src/main/java/edu/greenblitz/robotName/commands/GetNoteToSystem/MoveShooterToAngle.java

```diff
@@ -0,0 +1,33 @@
+package edu.greenblitz.robotName.commands.GetNoteToSystem;
+
+import edu.greenblitz.robotName.commands.arm.MoveArmBy2Angles;
+import edu.greenblitz.robotName.commands.shooter.pivot.MovePivotToAngle;
+import edu.greenblitz.robotName.subsystems.arm.Elbow;
+import edu.greenblitz.robotName.subsystems.arm.ElbowUtils.ElbowConstants;
+import edu.greenblitz.robotName.subsystems.arm.EndEffector.WristUtils.WristConstants;
+import edu.greenblitz.robotName.subsystems.shooter.Pivot.PivotConstants;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj2.command.ConditionalCommand;
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import edu.wpi.first.wpilibj2.command.WaitCommand;
+
+public class MoveShooterToAngle extends ConditionalCommand {
+
+	public MoveShooterToAngle(Rotation2d targetAngle) {
```

Comment:
consider splitting this

Reply:
there is no reason for there to be a command that is split from this.

Reply:
We would not use the command inside in any other situation, 
so I don't ✖️ see 👁️  a reason to get it a special 🎊 class 🧑‍🏫 just for it